lib/postprocessing/image.js

```diff
@@ -459,6 +469,35 @@ async function imagePostProcess(intermediateRendition, rendition, directories) {
     await write(img, rendition.path);
 }
 
+async function convertSvg(path, instructions) {
+
+    // IM automatically keeps aspect ratio
+    const width = instructions.width || 400;
+    const height = instructions.height || 400;
+    const fmt = instructions.fmt || "png";
+
+    // Creates intermediate image file since IM operation order matters
+    // Note RSVG is the converter here currently 2020/10/22 causing differences between normal Imagemagic use
+    const image = await read(path);
+
+    if (fmt === "jpg") {
```

Comment:
I know we normalize `jpeg` to `jpg`, but... Better be safe.
```suggestion
    if (fmt === "jpg" || fmt === "jpeg") {
```